ocf_data_sampler/torch_datasets/pvnet_uk_regional.py

```diff
@@ -423,22 +451,6 @@ def compute(xarray_dict: dict) -> dict:
     return xarray_dict
 
 
-def get_locations(ga_gsp: xr.DataArray) -> list[Location]:
```

Comment:
is this just not needed now? i cant remember what is was needed for?

Reply:
No its not used anymore. `get_gsp_locations()` is used instead. I just forgot to delete this when that new function was added